app/_src/production/secure-deployment/certificates.md

```diff
@@ -203,15 +201,21 @@ KUMA_API_SERVER_HTTPS_TLS_CERT_FILE=/tmp/tls.crt \
 {% endtab %}
 {% endtabs %}
 
-3) Configure secure connection using `kumactl` CLI tool:
-```sh
-kumactl config control-planes add \
-  --name=<NAME> \
-  --address=https://<KUMA_CP_DNS_NAME>:5682 \
-  --ca-cert-file=/tmp/ca.crt \
-```
+  If you recieve an error like the following, make sure you are using a supported certificate type and that the certificate doesn't contain incomplete or corrupted data:
```

Comment:
Do you know what cert types are supported? I'd like to list them here.

Reply:
From what I can see it's only PEM - https://github.com/kumahq/kuma/blob/fc86a9d3ae1fa14fe0a223dc74103a4948f8b83e/app/kuma-dp/pkg/dataplane/envoy/remote_bootstrap.go#L69